README.md

```diff
@@ -4,21 +4,20 @@
 
 Intel staging area for llvm.org contribution.
 Home for Intel LLVM-based projects:
- - SYCL* Compiler and Runtimes - compiler and runtime libraries for SYCL ([https://www.khronos.org/sycl/](https://www.khronos.org/sycl/)). See **sycl** branch.
+ - oneAPI Data Parallel C++ compiler - see **sycl** branch. More information on oneAPI and DPC++ is available at ([https://www.oneapi.com/](https://www.oneapi.com/))
```

Comment:
Could you re-format the text in this document to keep 80-character limit, please?

Reply:
np